gruntfile.js

```diff
@@ -62,73 +62,73 @@ module.exports = function(grunt) {
     },
```

Comment:
should probably be reverted?

Reply:
I think this was my mistake, when commiting some minor changes, accidentally included the gruntfile, that was then reverted. Just check if it's all okay.